4262 doc update silx compare doc (#4627)
silx-kit/silx: doc/source/applications/compare.rst

The *silx compare* command provides a graphical user interface to compare 2D data from files.

> This tools allow you to list list and open multiple datasets.

Reviewer: ```suggestion
This tool allows you to list land open multiple datasets.
```

This list can be updated with drag and drop from silx view.
This tools allow you to list list and open multiple datasets.

.. |imgViewImg| figure:: img/silx-compare.png
  :height: 300px
  :align: center

.. |imgViewImg| image:: img/silx-compare.png
   :height: 300px

>    :align: middle

Author: the `image`  routine cannot handle the `middle` alignment. So moved to `figure` -> `center`. More consistent with existing as well.

Vertical Compare Mode
"""""""""""""""""""""

- Splits the display horizontally into two panels
- Dataset ``A`` is displayed on the **left** side

> - Dataset ``B`` is displayed on the **right** side

Reviewer: ```suggestion
Splits the display horizontally into two panels:

- Dataset ``A`` is displayed on the **left** side
- Dataset ``B`` is displayed on the **right** side
```

🤷

Horizontal Compare Mode
"""""""""""""""""""""""

- Splits the display vertically into two panels
- Dataset ``A`` is displayed at the **top**

> - Dataset ``B`` is displayed at the **bottom**

Reviewer: ```suggestion
Splits the display vertically into two panels: 

- Dataset ``A`` is displayed at the **top**
- Dataset ``B`` is displayed at the **bottom**
```

Color-Encoded Difference Modes
""""""""""""""""""""""""""""""

Blue/Red Compare Mode
'''''''''''''''''''''

- Visualizes differences between datasets using a blue/red color scheme
- Useful for identifying positive and negative deviations

Yellow/Cyan Compare Mode
''''''''''''''''''''''''

- Visualizes differences using a yellow/cyan color scheme
- Provides alternative color mapping for difference visualization

Raw Compare Mode
''''''''''''''''

- Displays the raw concatenated data without color encoding

> - Shows unprocessed dataset values for direct inspection

Reviewer: What about making this a bit more compact?

Author: https://github.com/silx-kit/silx/pull/4627/commits/58c5c5322e915e22486f14b80279e35d78224841 will fix it.

More
----

You can add files by dragging and dropping them from ``silx view``.

>   .. figure:: http://www.silx.org/doc/silx/img/silx_compare_drag_and_drop.gif

Author: Two points:

* This is probably overkilled but this is adding some substance :(
* At the moment the two .gif are stored at silx.org. This might be something we want to change in the future ... But at least this doesn't pollute the git project.
